src/Package/MSTest.TestAdapter.Enu.nuspec

```diff
@@ -23,7 +23,8 @@
 	<dependencies>
 	    <group targetFramework="netstandard1.3">
 			<dependency id="System.Diagnostics.TextWriterTraceListener" version="4.3.0" />
-		</group>      
+			<dependency id="System.Xml.ReaderWriter" version="4.3.0" />
```

Comment:
This seems to be something we just stumbled upon at runtime. Should we just add in all the packages we are dependent upon in the dependencies section and just build against them instead?